bundles/CoreBundle/Resources/translations/de.json

```diff
@@ -894,4 +894,4 @@
     "disable_split_view": "Disable Split View",
     "split_view_object_dirty_title": "Please confirm",
     "split_view_object_dirty_msg": "There are unsaved modifications. You will lose all changes. Are you sure?"
-}
\ No newline at end of file
+}
```

Comment:
Sorry, obviously there ware some changes in my IDE which do not belong to this PR. Somehow I cannot revert this extra line break...

Reply:
mine (PHPStorm) does that too, as soon as I open a file, it creates the line break.

Reply:
I even edited the file in github's browser UI but no success.
Perhaps this file can be regenerated with PO editor...

Reply:
I believe this is intentional: https://github.com/pimcore/pimcore/blob/master/.editorconfig#L10

Reply:
No, that's just for `*.php` files, not for `*.json`

Reply:
@brusch would be nice, if you could revert changes of de.json concerning this PR, thanks.

Reply:
@BlackbitNeueMedien Should be easier/quicker for you, I'd have to clone your repo first ...

Reply:
Ok, now it worked - who knows why I was not able to revert the change last time...